### PR TITLE
ACS-4501 Migrate to GHA

### PR DESCRIPTION
**Pull request by aonych**
Migrate from Travis to GHA.

**Note**: delete the Travis cron job once this is merged.

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (dsibilio) [inline .github/workflows/ci.yml:19]: Reminder to delete the scheduled Travis cron job after merging this.

File: .github/workflows/ci.yml
Comment (dsibilio) [inline .github/workflows/ci.yml:53]: There is an extra space between `-y` and `imagemagick`.

File: .github/workflows/ci.yml
Comment (dsibilio) [inline .github/workflows/ci.yml:54]: It would be nice to add back the comment that was in the `init.sh` file, explaining what's the purpose of this line.

File: .github/workflows/ci.yml
Comment (dsibilio) [inline .github/workflows/ci.yml:69]: Is this `!failure()` condition actually needed? I can't see any way of skipping the build and veracode jobs.

**Review by dsibilio — CHANGES_REQUESTED** [review]

**Review by dsibilio — APPROVED** [review]
LGTM, assuming it passes.

**Review by damianujma — APPROVED** [review]

**Pull request merged**
